### PR TITLE
Location telecom translator implementation

### PR DESCRIPTION
**Pull request by jecihjoy**
## Description of what I changed
Location telecom translation implementation

## Issue I worked on

see https://issues.openmrs.org/browse/FM2-31

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**.

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/LocationTelecomTranslatorImplTest.java [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+		contactPoint = new ContactPoint();
+	}
+	
+	@Test
+	public void toFhirResource_shouldTranslateUuidToId() {
+		locationAttribute.setUuid(LOCATION_ATTRIBUTE_UUID);
+		locationAttribute.setValue(LOCATION_ATTRIBUTE_VALUE);
+		contactPoint = translator.toFhirResource(locationAttribute);
+		assertNotNull(contactPoint.getId());
+		assertEquals(contactPoint.getId(), LOCATION_ATTRIBUTE_UUID);
```
Comment (corneliouzbett) [inline api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/LocationTelecomTranslatorImplTest.java]: Prefer this, It's more readable
```
assertThat(contactPoint.getId(), notNullValue())
assertThat(contactPoint.getId(), equalTo(LOCATION_ATTRIBUTE_UUID))
```

File: api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/LocationTelecomTranslatorImplTest.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+		assertNotNull(contactPoint.getValue());
+		assertEquals(contactPoint.getValue(), LOCATION_ATTRIBUTE_VALUE);
+	}
+	
+	@Test
+	public void toOpenmrsType_shouldTranslateIdToUuid() {
+		contactPoint.setId(CONTACT_POINT_ID);
+		contactPoint.setValue(CONTACT_POINT_VALUE);
+		locationAttribute = translator.toOpenmrsType(contactPoint);
+		assertNotNull(locationAttribute.getUuid());
```
Comment (corneliouzbett) [inline api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/LocationTelecomTranslatorImplTest.java]: `assertThat(locationAttribute.getUuid(), notNullValue())`

File: api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/LocationTranslatorImpl.java [hunk 3]
```diff
@@ -11,20 +11,45 @@
...
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
 
 @Component
 @Setter(AccessLevel.PACKAGE)
 public class LocationTranslatorImpl implements LocationTranslator {
 	
+	private static Log log = LogFactory.getLog(LocationAddressTranslatorImpl.class);
+
```
Comment (corneliouzbett) [inline api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/LocationTranslatorImpl.java]: Declared but not used? You should probably remove this

**Comment by jecihjoy** [comment]
@corneliouzbett I have been using **assertEquals**  and **assertNotNull** through out my code.
I just realized I missed some instructions in the read me file. I'll do a follow up commit with this specific change

File: omod/src/main/java/org/openmrs/module/fhir2/web/servlet/FhirRestServlet.java [hunk 4]
```diff
@@ -35,10 +35,10 @@
 public class FhirRestServlet extends RestfulServer {
```
Comment (ibacher) [inline omod/src/main/java/org/openmrs/module/fhir2/web/servlet/FhirRestServlet.java]: Can you remove this file from this PR? Thanks!

Comment (jecihjoy) [reply omod/src/main/java/org/openmrs/module/fhir2/web/servlet/FhirRestServlet.java]: done 👍

**Pull request merged**